code/modules/medical/genetics/geneticsMachines.dm

```diff
@@ -509,6 +509,8 @@
 			if (!istype(H) || isprematureclone(H))
 				return
 			var/datum/bioEffect/mutantrace/BE = locate(params["ref"])
+			if (!H.mutantrace?.genetics_removable)
+				return
```

Comment:
probably good if we output to the user to let them know they can't remove this instead of silently failing
UI notif would be better but would be way more complex

Reply:
I've added it as a scanner_alert for now, if anyone wants to write UI code for it please do